tools/Docs/az-ps-latest.csv

```diff
@@ -0,0 +1,123 @@
+pac0,[ps=true;customSource=https://www.powershellgallery.com/api/v2/],Az.Accounts,2.5.1
```

Comment:
So this CSV does not have a header row, does it?

Reply:
Yes, no header row